install.sh

```diff
@@ -1,32 +1,21 @@
 #!/bin/sh -e
 
-WORKSPACE="${GITHUB_WORKSPACE:=/usr/local}"
 WORKSPACE="/usr/local"
-VERSION=${version:=v0.1.0-alpha.2}
 REPO=${repo:=open-component-model/ocm}
-URL=https://github.com/$REPO/releases/download
-PLATFORM=linux-amd64
-if [ "$REPO" != "gardener/component-cli" ]; then
-  ARCHIVESUFFIX=.tgz
-  ARCHIVEFILE="ocm-$PLATFORM$ARCHIVESUFFIX"
-else
-  ARCHIVESUFFIX=.gz
-  ARCHIVEFILE="componentcli-$PLATFORM$ARCHIVESUFFIX"
-fi
-FILE="$(basename "$ARCHIVEFILE" $ARCHIVESUFFIX)"
+VERSION=${version:=v0.2.0-rc.1}
+PLATFORM=Linux_x86_64
+ARCHIVESUFFIX=.tar.gz
+ARCHIVEFILE="ocm_$PLATFORM$ARCHIVESUFFIX"
```

Comment:
The formatting for the archive file's name changed to `"{{ .Binary }}-{{ .Version }}-{{ .Os }}-{{ .Arch }}"` as per @mandelsoft's feedback here: https://github.com/open-component-model/ocm/pull/291#discussion_r1132213989